content/en/docs/components/pipelines/reference/kfp-kubernetes.md

```diff
@@ -0,0 +1,9 @@
++++
+title = "Kubernetes Platform-specific Features"
+description = "Reference documentation for the `kfp-kubernetes` Python library"
+weight = 6
+                    
++++
+
+The `kfp-kubernetes` Python library enables authoring Kubeflow pipelines with Kubernetes-specific features. These features are supported by the default KFP open source BE. 
```

Comment:
```suggestion
The `kfp-kubernetes` Python library enables authoring Kubeflow pipelines with Kubernetes-specific features.
```

Reply:
agree, the sentence is really not needed